Gems/AtomLyIntegration/CommonFeatures/Code/Source/Material/MaterialBrowserInteractions.cpp

```diff
@@ -37,17 +39,19 @@ namespace AZ
         {
             if (AZStd::wildcard_match("*.material", fullSourceFileName))
             {
-                openers.push_back({ "Material_Editor", "Open in Material Editor...", QIcon(),
+                openers.push_back({ "Material_Editor", "Open in Material Editor...", QIcon(":/Menu/material_editor.svg"),
                     [&](const char* fullSourceFileNameInCallback, [[maybe_unused]] const AZ::Uuid& sourceUUID)
                     {
                         EditorMaterialSystemComponentRequestBus::Broadcast(
                             &EditorMaterialSystemComponentRequestBus::Events::OpenMaterialEditor,
                             fullSourceFileNameInCallback);
                     } });
             }
-            if (AZStd::wildcard_match("*.materialcanvas.azasset", fullSourceFileName))
+            if (AZStd::wildcard_match("*.materialgraph", fullSourceFileName) ||
+                AZStd::wildcard_match("*.materialgraphnode", fullSourceFileName) ||
+                AZStd::wildcard_match("*.shader", fullSourceFileName))
```

Comment:
Perhaps centralize these in a similar way to these: `AZ::Render::LightingPreset::Extension`, I see them used in a few places

Reply:
Updated